acquia/mc-cs-plugin-custom-objects
# Maut 2919

https://backlog.acquia.com/browse/MAUT-2919

#### Description:
This PR adds searching in lists on contact and custom item detail pages.

#### Steps to test this PR:
1. Create some custom object items.
2. Create some links for both custom items and contacts.
3. Go to the detail page of custom item that has some linked items and you should be able to filter the items the same way as in the custom object listing.
4. Go to the detail page of contact that has some linked items and you should be able to filter the items the same way as in the custom object listing.
